File: sdk/go/grpc/create_tenant.go
package main

import (
	"context"
	"log"

	v1 "github.com/Permify/permify-go/generated/base/v1"
	permify "github.com/Permify/permify-go/v1"
	"google.golang.org/grpc"
	"google.golang.org/grpc/credentials/insecure"
)

func main() {
	client, err := permify.NewClient(
		permify.Config{
			Endpoint: `localhost:3478`,
		},
		grpc.WithTransportCredentials(insecure.NewCredentials()),
	)
	if err != nil {
		log.Fatalf("failed to create Permify client: %v", err)
	}

	ct, err := client.Tenancy.Create(context.Background(), &v1.TenantCreateRequest{
		Id:   "t1",
		Name: "tenant 1",
	})
	if err != nil {
		log.Fatalf("failed to create tenant: %v", err)
	}

	log.Printf("Tenant created: %v", ct)
}

File: sdk/javascript/grpc/create_tenant.js
const permify = require("@permify/permify-node");

const request = new permify.grpc.payload.TenantCreateRequest();
request.setId("t1");
request.setName("tenant 1");

const client = permify.grpc.newClient({
    endpoint: "localhost:3478",
    cert: undefined
});

client.tenancy.create(request).then((response) => {
    console.log(response);

    // handle response
})